src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -25,6 +25,7 @@ public enum Language
         PL,
         PT,
         RO,
+        RS,
```

Comment:
```suggestion
        SR
```
Here and everywhere else.

Reply:
OMG.. I was fooled from this one:
https://en.wikipedia.org/wiki/Serbia
![image](https://user-images.githubusercontent.com/13002713/204033160-af391bc3-ee63-4039-a6ee-785245d3a0bd.png)
But wrong ISO code.
This one is ok:
https://en.wikipedia.org/wiki/Serbian_language

Reply:
Fixed